hbase-examples/src/main/java/org/apache/hadoop/hbase/util/ClientUtils.java

```diff
@@ -110,4 +109,13 @@ public static String utf8(final byte[] buf) {
     }
   }
 
+  /**
+   * Helper to translate a byte buffer to UTF8 strings
+   * @param buf byte buffer
+   * @return UTF8 decoded string value
+   */
+  public static String utf8(final ByteBuffer buf) {
+    return buf.toString();
```

Comment:
I do not think this is what we want? We should copy the content a byte array and then call the above utf8 method?